bazel-java-deps.bzl

```diff
@@ -10,15 +10,15 @@ def install_java_deps():
             "args4j:args4j:2.33",
             "ch.qos.logback:logback-classic:1.2.3",
             "ch.qos.logback:logback-core:1.2.3",
-            "com.auth0:java-jwt:3.8.1",
-            "com.auth0:jwks-rsa:0.9.0",
+            "com.auth0:java-jwt:3.8.2",
+            "com.auth0:jwks-rsa:0.8.3",
             "com.chuusai:shapeless_2.12:2.3.2",
-            "com.fasterxml.jackson.core:jackson-annotations:2.9.8",
-            "com.fasterxml.jackson.core:jackson-core:2.9.8",
-            "com.fasterxml.jackson.core:jackson-databind:2.9.8",
-            "com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:2.9.8",
-            "com.fasterxml.jackson.module:jackson-module-parameter-names:2.9.8",
-            "com.fasterxml.jackson.module:jackson-module-scala_2.12:2.9.8",
+            "com.fasterxml.jackson.core:jackson-annotations:2.9.0",
```

Comment:
For consistency, you might want to depend on 2.9.9 for jackson-annotations as well.

Reply:
@miklos-da i believe mirrored what auth0 had as their dependencies defined in the maven central pom here - i believe even jackson itself is inconsistent in which versions it uses.  but will double-check to make sure

Reply:
yeah just double-checked -- jackson databind 2.9.9.3 actually depends on annotations 2.9.0 in maven central so matched that to avoid diverging . https://mvnrepository.com/artifact/com.fasterxml.jackson.core/jackson-databind/2.9.9.3
![image](https://user-images.githubusercontent.com/25179017/70436238-cdc57180-1a56-11ea-80ba-210a631215ce.png)

Reply:
Thanks for double-checking that.